README.md

```diff
@@ -44,6 +44,7 @@ The following options are supported:
 
 | **Option**                   | **Data Type** | **Default value**                          | **Description** |
 | ---------------------------- | ------------- | ------------------------------------------ |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bands`                      | array of numbers | undefined | an array mapping the bands to the output bands. must be of length 3 or 4. |
```

Comment:
I'd suggest making the one-band use case a bit simpler by allowing 1 and 2 elements, too.
- 1 element means single band w/o alpha (i.e. expand from e.g. [0] to [0,0,0])
- 2 elements mean single band with alpha (i.e. expand from e.g. [0, 1] to [0,0,0,1])
- 3 and 4 as defined below.

```suggestion
| `bands`                      | array of numbers | undefined | an array mapping the bands to the output bands. must be of length 1, 2, 3 or 4. |
```

Reply:
=> #57

Reply:
That's a great idea!